lib/matplotlib/font_manager.py

```diff
@@ -1366,7 +1366,12 @@ def is_opentype_cff_font(filename):
 _fmcache = None
 
 
-get_font = lru_cache(64)(ft2font.FT2Font)
+_get_font = lru_cache(64)(ft2font.FT2Font)
+
+def get_font(filename, hinting_factor=None):
+    if hinting_factor is None:
+        hinting_factor = rcParams['text.hinting_factor']
+    return _get_font(filename, hinting_factor)
```

Comment:
Is not it equals to?
```python
@lru_cache(64)
def get_font(filename, hinting_factor=None):
    if hinting_factor is None:
        hinting_factor = rcParams['text.hinting_factor']
    return ft2font.FT2Font(filename, hinting_factor)
```

Reply:
That would cache `hinting_factor=None` with whatever the value of the rcParam was at first call.